worker/src/batch_maker.rs

```diff
@@ -90,6 +88,18 @@ impl<PublicKey: VerifyingKey> BatchMaker<PublicKey> {
                     }
                     timer.as_mut().reset(Instant::now() + self.max_batch_delay);
                 }
+
+                // Trigger reconfigure.
+                result = self.rx_reconfigure.changed() => {
+                    result.expect("Committee channel dropped");
+                    let message = self.rx_reconfigure.borrow().clone();
+                    match message {
+                        Reconfigure::NewCommittee(new_committee) => {
+                            self.committee=new_committee;
+                        },
+                        Reconfigure::Shutdown(_token) => return
```

Comment:
I suppose the `Shutdown` message is more uniform than cancelling the task (since you don't have to go fetch the handle and can just remember the channel). The later will be more familiar to most, though: do we have a few lines of documentation somewhere about our chosen discipline?